.github/workflows/ci.yml

```diff
@@ -708,63 +721,6 @@ jobs:
             --trino-jdk-version zulu_${{ matrix.jdk }}
       - name: Product Tests (impacted-features)
```

Comment:
Do you still need this step? Looks like it does the same `suite run`  as the `Product Tests (all)` suite

Reply:
Good catch. It might have been an issue when merging things. This run should pass `--impacted-features impacted-features.log` to ptl.

Reply:
Yup, good catch, thanks! I'll rebuild this step to use an env var so it's easier to read and to avoid similar mistakes in the future.